tardis/analysis/opacities.py

```diff
@@ -329,7 +329,7 @@ def _calc_thomson_scattering_opacity(self):
         """Calculate the Thomson scattering opacity for each grid cell
 
         \[
-          \chi_{\mathrm{Thomson}} = n_{\mathrm{e}} \sigma_{\mathrm{T}}
+        \chi_{\mathrm{Thomson}} = n_{\mathrm{e}} \sigma_{\mathrm{T}}
```

Comment:
Why is this changed?

Reply:
Just for formatting consistency with Docstrings 😄